test/packaging/test.sh

```diff
@@ -11,4 +11,4 @@ if [ "$NR_LICENSE_KEY" = "" ]; then
 fi
 
 printf "\nTesting initial install...\n"
-NR_LICENSE_KEY="$NR_LICENSE_KEY" ansible-playbook -i "$ANSIBLE_INVENTORY" test/packaging/ansible/test.yml
```

Comment:
remove `-v` please
also path should be there to be able to run from root folder